build_docs.pl

```diff
@@ -127,18 +127,69 @@ sub build_local {
             exit;
         }
         else {
-            my $http = dir( 'resources', 'http.py' )->absolute;
-            close STDIN;
-            open( STDIN, "</dev/null" );
-            chdir $dir;
-            exec( $http '8000' );
+            if ( $running_in_docker ) {
+                # We use nginx to serve files instead of the python built in web server
+                # when we're running inside docker because the python web server performs
+                # badly there. nginx is fine.
```

Comment:
The python web server is fine some of the time but hangs for many seconds at a time. I *suspect* it'll do that on some Linux systems as well, but nginx runs just fine here.